model/task/db.go

```diff
@@ -1121,11 +1121,12 @@ func FindWithDisplayTasks(query db.Q) ([]Task, error) {
 		return nil, nil
 	}
 
-	for _, t := range tasks {
+	for i, t := range tasks {
 		_, err = t.GetDisplayTask()
 		if err != nil {
 			return nil, errors.Wrap(err, "unable to retrieve parent display task")
 		}
+		tasks[i] = t
```

Comment:
did this function not work before ?? feeling shook

Reply:
same -_- this makes me doubt that all the callers of this actually need this overhead